controllers/src/test/scala/cromwell/pipeline/controller/RunControllerTest.scala

```diff
@@ -49,47 +49,44 @@ class RunControllerTest
         when(runService.getRunByIdAndUser(runId, accessToken.userId))
           .thenReturn(Future.failed(new RuntimeException("something went wrong")))
 
-        Get("/runs?run_id=" + runId.unwrap) ~> runController.route(accessToken) ~> check {
+        Get("/runs/" + runId.unwrap) ~> runController.route(accessToken) ~> check {
           status shouldBe StatusCodes.InternalServerError
         }
       }
 
       "returns run entity if the run id is not valid" taggedAs Controller in {
         val invalidRunId = "123"
-        Get("/runs?run_id=" + invalidRunId) ~> runController.route(accessToken) ~> check {
-          rejection shouldBe MalformedQueryParamRejection("run_id", "Invalid RunId")
+        Get("/runs/" + invalidRunId) ~> Route.seal(runController.route(accessToken)) ~> check {
+          status shouldBe StatusCodes.NotFound
         }
       }
     }
 
     "deleteRunById" should {
       "returns run 1 if the entity was deleted" taggedAs Controller in {
         val runId = TestRunUtils.getDummyRunId
-        val deleteRequest = RunDeleteRequest(runId)
 
         when(runService.deleteRunById(runId, accessToken.userId)).thenReturn(Future.successful(1))
 
-        Delete("/runs", deleteRequest) ~> runController.route(accessToken) ~> check {
+        Delete("/runs/" + runId.unwrap) ~> runController.route(accessToken) ~> check {
           status shouldBe StatusCodes.OK
         }
       }
       "returns server error if run delete was failed" taggedAs Controller in {
         val runId = TestRunUtils.getDummyRunId
-        val deleteRequest = RunDeleteRequest(runId)
         when(runService.deleteRunById(runId, accessToken.userId))
           .thenReturn(Future.failed(new RuntimeException("something went wrong")))
 
-        Delete("/runs", deleteRequest) ~> runController.route(accessToken) ~> check {
+        Delete("/runs/" + runId.unwrap) ~> runController.route(accessToken) ~> check {
```

Comment:
Actually we don't need `.unwrap` everywhere. `s"/runs/$runId"` should work just fine

Reply:
Hmmm. It`s really work. Thanks.

Reply:
Changed.